packages/fxa-content-server/app/tests/spec/views/choose_what_to_sync.js

```diff
@@ -128,30 +129,62 @@ describe('views/choose_what_to_sync', () => {
         assert.isTrue(view.navigate.calledWith('signup'));
       });
     });
+  });
 
-    it('renders email info, adds SCREEN_CLASS to body', () => {
-      return initView().then(() => {
-        assert.include(view.$('.success-email-created').text(), email);
+  describe('pollVerification', () => {
+    it('polls', () => {
+      model.set('pollVerification', true);
+      return initView()
+        .then(() => {
+          sinon.spy(view, 'waitForSessionVerification');
+          return view.afterVisible();
+        })
+        .then(() => {
+          assert.isTrue(view.waitForSessionVerification.calledOnce);
+        });
+    });
 
-        assert.isTrue($('body').hasClass(View.SCREEN_CLASS));
+    it('does not poll', () => {
```

Comment:
When does this happen?

Reply:
same scenario as https://github.com/mozilla/fxa/pull/2534#discussion_r329639774 , I spoke to @ryanfeeley and for now with email verification we do not poll when there's a choice of not syncing

Reply:
same scenario as #2534 (comment) , I spoke to @ryanfeeley and for now with email verification we do not poll when there's a choice of not syncing